src/build/ci/find-unapproved-junit.sh

```diff
@@ -38,8 +36,10 @@ function findalljunitproblems() {
   fi
   # find any new classes using something other than the jupiter API, except those allowed
   grep "$opts" --include='*.java' 'org[.]junit[.](?!jupiter)' | grep -Pv "^(${ALLOWED_PIPE_SEP//./[.]})\$"
-  # find any uses of the jupiter API in the allowed vintage classes
-  grep "$opts" 'org[.]junit[.]jupiter' "${ALLOWED[@]}"
+  if ((${#ALLOWED[@]} != 0)); then
```

Comment:
Could be a follow on issue, wondering if the ALLOWED set should just be removed from the script since its empty.  Would simplify the script.

Reply:
I left it there just in case we needed it in the future for some reason. We have the same thing in accumulo-access with this script.